packages/help-center/src/hooks/use-open-article-in-help-center.ts

```diff
@@ -0,0 +1,23 @@
+import { HelpCenter } from '@automattic/data-stores';
+import { useDispatch as useDataStoreDispatch } from '@wordpress/data';
+
+const HELP_CENTER_STORE = HelpCenter.register();
+
+export const useOpenArticleInHelpCenter = () => {
+	const helpCenterDispatch = useDataStoreDispatch( HELP_CENTER_STORE );
+	const setShowHelpCenter = helpCenterDispatch?.setShowHelpCenter;
+	const setNavigateToRoute = helpCenterDispatch?.setNavigateToRoute;
+
+	const openArticleInHelpCenter = ( articleLink: string ) => {
+		if ( setShowHelpCenter && setNavigateToRoute ) {
+			const articleRoute = `/post/?link=${ articleLink }`;
+			setNavigateToRoute( articleRoute );
+			setShowHelpCenter( true );
```

Comment:
Perhaps I can refactor `setShowSupportDoc` and use it instead of `setNavigateToRoute` and `setShowHelpCenter` 🤔

Reply:
It might be a valid idea, let's try this!

Reply:
@escapemanuele - I updated the setShowSupportDoc function and it works for this use case as well. Let me know if you have any suggestions for the minor change I made.